modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/MessagingEngine.java

```diff
@@ -524,6 +524,20 @@ public long getMessageCountOfQueue(String queueName) throws AndesException {
     }
 
     /**
+     * Get number of messages in the queue withing the message id range
```

Comment:
Yep, It's a word [1]. But that's not what I meant though ;) Will fix.

[1] https://www.google.com/search?client=ubuntu&channel=fs&q=withing&ie=utf-8&oe=utf-8#channel=fs&q=withing+meaning